_posts/2025-04-22-OpenSearch-version-3-features.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title:  "OpenSearch® version 3.0: New features, breaking changes from Apache Lucene™ 10, and what to expect"
+authors:
+ - kwren
+date: 2025-04-22
```

Comment:
Let's change it to 2025-04-24 - assuming that all edits are incorporated by that date.